_delphi_utils_python/delphi_utils/archive.py

```diff
@@ -621,46 +621,13 @@ def update_cache(self):
         self._cache_updated = True
 
 if __name__ == "__main__":
-    parser = ArgumentParser()
-    parser.add_argument("--archive_type", required=True, type=str,
-                        choices=["git", "s3", "filesystem"],
-                        help="Type of archive differ to use.")
-    parser.add_argument("--indicator_prefix", type=str, default="",
-                        help="The prefix for S3 keys related to this indicator."
-                        " Required for `archive_type = 's3'")
-    parser.add_argument("--branch_name", type=str, default="",
-                        help=" Branch to use for `archive_type` = 'git'.")
-    parser.add_argument("--override_dirty", action="store_true",
-                        help="Whether to allow overwriting of untracked &"
-                        " uncommitted changes for `archive_type` = 'git'")
-    parser.add_argument("--commit_partial_success",  action="store_true",
-                        help="Whether to still commit for `archive_type` = "
-                        "'git' even if some files were not archived and "
-                        "staged due to `override_dirty` = False.")
-    parser.add_argument("--commit_message", type=str, default="",
-                        help="Commit message for `archive_type` = 'git'")
-    args = parser.parse_args()
-    params = read_params()
+    _params = read_params()
 
     # Autodetect whether parameters have been factored hierarchically or not
     # See https://github.com/cmu-delphi/covidcast-indicators/issues/847
     # Once all indicators have their parameters factored in to "common", "indicator", "validation",
     # and "archive", this code will be obsolete.
-    if "archive" in params:
-        archive_params = params["archive"]
-        common_params = params["common"]
-    else:
-        archive_params = params
-        common_params = params
-
-    run_module(args.archive_type,
-               archive_params["cache_dir"],
-               common_params["export_dir"],
-               aws_credentials=archive_params.get("aws_credentials", {}),
-               branch_name=args.branch_name,
-               bucket_name=archive_params.get("bucket_name", ""),
-               commit_message=args.commit_message,
-               commit_partial_success=args.commit_partial_success,
-               indicator_prefix=args.indicator_prefix,
-               override_dirty=args.override_dirty
-               )
+    if "archive" not in _params:
```

Comment:
what's this doing? if this evals to true, shouldn't the archiver not run?

Reply:
The thought here is that if you are explicitly calling the module from the command line, you intend to run archiving.  If we can't find "archive" in the params, we are interpreting that as "this params.json has not been refactored to use the new style" (see #847 and the old version of this).

Reply:
ah got it, thanks. can you leave a comment explaining the old structure bit?

Reply:
Done